backend/routes/tweets.js

```diff
@@ -0,0 +1,15 @@
+var express = require('express');
+var router = express.Router();
+const pool = require("../db");
+
+router.get('/', async(req, res) => {
+    try {
+        const allTweets = await pool.query("SELECT * FROM twitter");
```

Comment:
I wish you had done this in promise handling style like real men do but ok.

Reply:
bru uncull